chia/full_node/mempool_check_conditions.py

```diff
@@ -10,9 +12,9 @@
 from chia.types.condition_with_args import ConditionWithArgs
 from chia.types.generator_types import BlockGenerator
 from chia.types.name_puzzle_condition import NPC
-from chia.util.clvm import int_from_bytes
+from chia.util.clvm import int_from_bytes, int_to_bytes
 from chia.util.condition_tools import ConditionOpcode, conditions_by_opcode
-from chia.util.errors import Err
+from chia.util.errors import Err, ValidationError
```

Comment:
to not disrupt the return paths to introduce an error code, I throw `ValidationError`, which includes an error code. It seems to exist for this purpose.

Reply:
theres a msg field in validation error if you need to differentiate similar errors, but it seems we are not printing it in receive_block anymore